src/platform-includes/upload-dif/dotnet.mdx

```diff
@@ -0,0 +1,4 @@
+```bash
+sentry-cli login
+sentry-cli upload-dif -o {YOUR ORGANISATION} -p {PROJECT} bin/Release
```

Comment:
@mattjohnsonpint do we need to upload from `obj/Release` instead? or is the path here completely different?